templates/skeleton/contrib/qooxdoo.json

```diff
@@ -0,0 +1,8 @@
+{
+	"libraries": [
+		{ "path": "." }
+	],
+	"applications": [
```

Comment:
is the applications key here so that applications that would previously have shipped as part of a contrib (eg a demo/showcase app) can be separate to the contrib?

Reply:
The separation is between libraries which can be included into other projects and applications which can be compiled and run. Both are technically libraries, but qooxdoo-contrib is only looking for the former type of library. The latter type is there so that we can, for example, automatically build showcases for contribs. `qooxdoo.json` lets the developer decide which libraries and applications to expose to our system. We could extend the meta-data to contain more information than just the path. But then again it might be better to be very conservative here:  in general, the Manifest is the right place for metadata. Otherwise we risk to have to maintain a Meta-Manifest on top of the other Manifests.